linkchecker

```diff
@@ -578,7 +579,7 @@ if options.output:
     if "/" in options.output:
         logtype, encoding = options.output.split("/", 1)
     else:
-        logtype, encoding = options.output, i18n.default_encoding
+        logtype, encoding = options.output, linkcheck.i18n.default_encoding
```

Comment:
Same here: since we're not explicitly importing `linkcheck.i18n` here any longer, we now rely on `linkcheck/__init__.py` having an `import .i18n`.

Reply:
As we are relying on `__init__.py` to call `i18n.init()` I think this is OK.